plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesPublisher.java

```diff
@@ -243,4 +277,64 @@ private QualityGateEvaluationMode determineQualityGateEvaluationMode(final Repor
         }
         return qualityGateEvaluationMode;
     }
+
+    private String extractChecksSummary(final IssuesStatistics statistics) {
+        return String.format("## %d issues in total:\n"
+                        + "- ### %d new issues\n"
+                        + "- ### %d Outstanding Issues\n"
```

Comment:
the case should be consistent for each section
```suggestion
                        + "- ### %d outstanding Issues\n"
```